### PR TITLE
Aetolia wings and doors

### PR DESCRIPTION
**Pull request by mccarp**
Added aetolia wings and caves travel items.
This includes a gagged survey which runs each time you change area to track your current continent.

Added aetolia lines to door triggers

### CODE REVIEW

**Review by vadi2 — COMMENTED** [review]
👍 code checks out to me. Have anyone to test the changes?

**Comment by mccarp** [comment]
With further testing, it was found out that off-plane locations don't have continent data on survey. I adjusted the survey trigger to also capture the plane to handle these cases. I have tested it and it seems to work okay, and I had @lucentlady help test it too.

**Review by luculentlady — APPROVED** [review]
I tested the code on Aetolia, and it ran smoothly.

**Comment by vadi2** [comment]
Alright and why the move for the options table? It makes it very hard to see what was actually changed (if anything), when looking at the diff.

**Comment by mccarp** [comment]
The reason I needed to move the option table was because of this:
`
function mmp.createOption(startingValue, onChangeFunc, allowedVarTypes, use, checkOption, games) .... end
`
With the option table above the load settings script, when mudlet loads it would be trying to create this function before the mmp namespace was initialized, and it would error / not work properly. For some reason it didn't have any problems when loaded as a package, but it did when as a module.

**Comment by vadi2** [comment]
That warrants a closer look - mind investigating what exactly is the difference? There should not be one

**Comment by demonnic** [comment]
so, I've looked into this a little bit, and when you first load mudlet-mapper.xml as a package it errors, but then it does not error after closing and reopening the profile. Closing and reopening Mudlet also does not error when installed as a package.

When installed as a module with priority 0, it fails every time it is loaded. Looking over the package, I can see no reason it should ever work as a package, as it's definitely trying to access the mmp table before it's been instantiated. I've done a search for anywhere else it might be getting set, and verified it isn't listed as a saved variable in the package, nor is the checkbox for saving the variable checked in the variable tab. I have to wonder if this is to do with the way a package is folded into the overall profile but a module is reloaded from scratch each time. Except I still don't know how that would make the package version function. But if I do `lua resetProfile()` then the errors come back, even when installed as a package. So the only time it -does- work with the current version is if it's installed as a package when the profile is loaded, and the profile is not reset using resetProfile(). Feels like a weird corner case in Mudlet itself.

This is all using the version of the xml from the primary branch of IRE-Mudlet-Mapping. Using the version proposed in this PR it loads fine either way. which is what I would expect. I may try to see if I can recreate this with a minimal package.

**Comment by vadi2** [comment]
Okay. Could also file an issue for it?

**Review by vadi2 — APPROVED** [review]

**Comment by vadi2** [comment]
@keneanung are you good with this as well?

**Comment by demonnic** [comment]
issue opened upstream with Mudlet: https://github.com/Mudlet/Mudlet/issues/6460

File: mudlet-mapper.xml [hunk 1]
```diff
@@ -10597,6 +10752,24 @@ end</script>
...
+					<Script isActive="yes" isFolder="no">
+						<name>mmp.aetContinentCheck</name>
+						<packageName></packageName>
+						<script>local oldArea = oldArea or ""
+function mmp.aetContinentCheck()
+  if mmp.game ~= "aetolia" then return end
+  
+  if oldArea ~= gmcp.Room.Info.area then
+     oldArea = gmcp.Room.Info.area
+     enableTrigger("Aetolia Gag Survey")
```
Comment (keneanung) [inline mudlet-mapper.xml:10764]: Minor indentation suggestion
```suggestion
    oldArea = gmcp.Room.Info.area
    enableTrigger("Aetolia Gag Survey")
```

**Review by keneanung — APPROVED** [review]
Looks fine to me with one minor indentation suggestion below

**Pull request merged**